### PR TITLE
Addressed review from @sindresorhus 

### LINKED ISSUE #1861: Address review from @sindresorhus
There are some awesome ideas in this tweet: https://twitter.com/sindresorhus/status/1359226274368028672

We need to:
- fix `python` -> `Python`
- Describe why "strict" is cool 💪 
- Add more examples of bad code => good code

### PR DESCRIPTION
**Pull request by aceking007**
# I have updated documentation!

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [ ] I have created at least one test case for the changes I have made
- [x] I have updated the documentation for the changes I have made
- [x] I have added my changes to the `CHANGELOG.md`

## Related issues

Format is:
- Closes #1861

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,17 +50,52 @@ And it won't change your existing workflow.
        alt="invocation results">
 </p>
 
+### Example
```
Comment (sobolevn) [inline README.md:53]: Let's remove this section for now:
1. I will make screenshots with the same style as "Running"
2. I will have to think about some cool examples to showcase

File: README.md [hunk 2]
```diff
@@ -50,17 +50,52 @@ And it won't change your existing workflow.
...
+The updated code is pretty readable and has a consistent structure that is lacking in the bad code. The linter also suggests to simplify the logic of the code which makes it less redundant.
+
 See ["Usage" section](https://wemake-python-stylegui.de/en/latest/pages/usage/setup.html)
-in the docs for examples and integrations.
+in the docs for more examples and integrations.
 
 We also support [GitHub Actions](https://wemake-python-stylegui.de/en/latest/pages/usage/integrations/github-actions.html) as first class-citizens.
 [Try it out](https://github.com/marketplace/actions/wemake-python-styleguide)!
 
+## Strict is the new cool
```
Comment (sobolevn) [inline README.md:85]: Amazing work! You nailed this part 🚀

File: README.md [hunk 3]
```diff
@@ -50,17 +50,52 @@ And it won't change your existing workflow.
...
+
+Strict linting offers the following benefits to developers and companies:
+1. Ensures consistency - no matter who works on it, the end product will always be the same dependable code
+2. Helps avoid potential bugs - strict rules make sure that you don't contradict/make a fool of yourself in the future
```
Comment (sobolevn) [inline README.md:89]:
```suggestion
2. Helps avoid potential bugs - strict rules make sure that you don't make common mistakes
```

File: README.md [hunk 4]
```diff
@@ -50,17 +50,52 @@ And it won't change your existing workflow.
...
+3. Efficient code reviews - each piece of code has a similar familiar style and syntax. If it passes all the checks, there's little left to review!
+4. Less code revisions - strict linting ensures that you don't have to re-write the codebase again and again
+5. Reduce code redundancy - Sometimes we write complex code as we are thinking in a certain way about a problem. The linter offers suggestions that can help simplify the code and eliminate redundant statements.
```
Comment (sobolevn) [inline README.md:92]:
```suggestion
5. Reduce code redundancy - Sometimes we write complex code as we are thinking in a certain way about a problem. The linter offers suggestions that can help simplify the code and eliminate redundant statements
```

**Review by sobolevn — CHANGES_REQUESTED** [review]
Thanks a lot! Great job!

The only major thing: let's remove "Example" section for now. We can work on it later on.
(Don't forget to delete static images as well 🙂 )

File: README.md (hunk 2 above)
Comment (aceking007) [reply README.md:85]: Thanks!

File: README.md (hunk 1 above)
Comment (aceking007) [reply README.md]: I have removed this section and I will push the PR again.  
Do check and let me know if this is fine.

**Review by sobolevn — APPROVED** [review]
Great work! 👍 

Thanks!

**Pull request merged**